.xcode-selective-testing.yml

```diff
@@ -0,0 +1,2 @@
+basePath: wire-ios-mono.xcworkspace
+testPlan: AllTests.xctestplan
```

Comment:
Configuration file for XcodeSelectiveTesting is not a necessity but a convenience. Same parameters can be passed via CLI

Reply:
Interesting tool, thanks! How does `XcodeSelectiveTesting` calculate the dependency graph? As far as I can see from the [Documentation](https://github.com/mikeger/XcodeSelectiveTesting?tab=readme-ov-file#2-build-the-dependency-graph) it uses SPM, but we have most dependencies in `Carthage`.

Reply:
Hey @markusfassbender , thanks for the reply! The tool has support for SPM packages and for Xcode projects. It's working well with Wire monorepo setup. I will probably create a short video to showcase the integration in the upcoming days.